babel.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   presets: [
     '@vue/app'
+  ],
+  plugins: [
+    ['@babel/plugin-proposal-class-properties', { loose: true }]
```

Comment:
So that we support static fields in classes with ES6. (`loose` so that it is less strict with how we define the static fields)